pandas/io/formats/style.py

```diff
@@ -1554,7 +1558,8 @@ def _update_ctx_header(self, attrs: DataFrame, axis: AxisInt) -> None:
                 else:
                     self.ctx_columns[(j, i)].extend(css_list)
 
-    def _copy(self, deepcopy: bool = False) -> Styler:
+    @classmethod
+    def _copy(cls, self, deepcopy: bool = False) -> Styler:
```

Comment:
Is it necessary to make this a class method? Although this is really only an internal method doing this completely changes its signature. Is it not sufficient to keep it as a default method and use only the `type(self)(..)` line below?

Reply:
Totally agree, I think I was just going off of the suggestion in the issue, but that makes total sense. Made the change, also makes the diff much cleaner!